### ISSUE TITLE
The Terrorism Acts

### ISSUE THREAD

**Issue by digitalWestie** [openpolitics/manifesto#320]
I'm not too clear on how the process works exactly, however I'm not a fan of the proscribed terror list where one day to the next you can criminalize almost anything and everything to do with an organisation. 

I became interested in the Kurdish resistance movement and the PKK not too long ago. When the heavy fighting took place around Kobane, many people like myself wanted to show solidarity. About then I discovered the PKK is on the terrorism list. Under terrorism acts uniform is an offence, so wearing/showing symbols can get you [lifted](http://www.legislation.gov.uk/ukpga/2000/11/part/II/crossheading/offences). 

I'd also encourage you to have a look at "support" and "organising" under act. Around that time, people rallied in George Square in Glasgow and Trafalgar in London, many of them bearing PYG and PKK flags and symbols. It's also interesting note the definition of 'meeting' under the act. This could criminalize a group of human rights lawyers meeting to discuss the situation of a group. I did read about something similar happening in the US one day but I've since lost the article. 

Needless to say, we don't live in a black and white world. Often we leave it up to our courts to decide the grey areas. But under the present 'terror list' system a handful of people in government are essentially dividing the world up according to the arbitrary lines that suit them.

**Comment by Floppy** [comment]
Agreed that terrorism acts need some reform, and give powers that seem too wide. Anyone expert enough to attempt some changes, or could point us at a group campaigning for change?

**Comment by digitalWestie** [comment]
Oh and remember this http://news.bbc.co.uk/1/hi/uk_politics/7688560.stm ?

Anyway these Article19 folk might be worth talking to http://www.article19.org/pages/en/what-we-do.html

**Comment by philipjohn** [comment]
I've thought about this a few The Green Party seem to have a sensible policy on this, I think, stating that it should [not be illegal to be a member of an organisation](http://www.bbc.co.uk/news/uk-politics-30973388), even if it's deemed to be a "terrorist".

> Ms Bennett said people should not be punished "for what they think or what they believe".

**Comment by digitalWestie** [comment]
Yeah, I think it raises a good point ie the distinction between being a member of an organisation vs actively financing, or facilitating crime in some way. It's highly likely they'd overlap but not necessarily.

**Comment by yellowgopher** [comment]
I agree.  However we need to consider whether being a member does give an organisation some currency (both figuratively and, maybe, physically!).  A belief (there's that word again) and a symbol are not necessarily dangerous (he says, slightly uncomfortably) but what we do with them can be!

**Comment by digitalWestie** [comment]
Yup, to be clear if you're funding acts of aggression that's a crime. I also see that incitement to crime or threats / abuse / harassment as crimes. 

However, I resist the notion that a statement or symbol is criminal per se, even if it's completely awful. 

For example, I don't think someone should come after you if you make a statement like "they were right to bomb X". 

The point is I don't believe in an arbitrary list of what is right and what is wrong. Hence more than anything my beef is more with the ease with which govt. can decide an organisation is 'terrorist'.
